core/modules/system/system.install

```diff
@@ -885,6 +885,26 @@ function system_schema() {
   $schema['cache_token'] = $schema['cache'];
   $schema['cache_token']['description'] = 'Cache table for token information.';
 
+  $schema['invalidated_cache_tags'] = array(
```

Comment:
Do we need an update hook too to add this table to existing sites?

Reply:
Yes, I have created an update hook now.